lib/metasploit/framework/login_scanner/mssql.rb

```diff
@@ -47,6 +47,8 @@ class MSSQL
         #   @return [Boolean] Whether to use Windows Authentication instead of SQL Server Auth.
         attr_accessor :windows_authentication
 
+        attr_accessor :use_client_as_proof
```

Comment:
Can we make sure we've added all of the yard doc types here - and other places:

https://github.com/rapid7/metasploit-framework/blob/b72d8d16d4e038fa57e0002af3c185cd190bc19c/lib/metasploit/framework/login_scanner/postgres.rb#L14-L16